docs/tutorial/forge-overview.md

```diff
@@ -0,0 +1,30 @@
+# Distributing Apps With Electron Forge
+
+Electron Forge is a complete tool for creating, publishing, and installing modern Electron applications. Electron Forge unifies the existing (and well maintained) build tools for Electron development into a simple, easy to use package so
+that anyone can jump right in to Electron development.
```

Comment:
:+1: on @erickzhao's suggestions, piggybacking on them here

```suggestion
Electron Forge is a tool for packaging and publishing Electron applications.
It unifies Electron's tooling ecosystem into a single extensible interface so
that anyone can jump right in to making Electron apps.
```

Minor changes:

- "modern" in "modern Electron applications" is implied
- "creating" is a little ambiguous; it _could_ be read as Forge also being an IDE. Changed to "packaging"
- 'Electron development" -> clarify that we're talking about app development, not Electron maintenance
- "Complete tool" - might just be me, but that phrasing didn't scan right to me

Reply:
I agree with both of you! Similar wording is used elsewhere in our documentation: I'll make changes there as well to modernize